src/Generator/EpubGenerator.php

```diff
@@ -339,6 +340,7 @@ private function getXhtmlTitle( Book $book ) {
 			$footerElements[] = $book->year;
 		}
 
+		$formatter = new IntlDateFormatter( $book->getLocale(), IntlDateFormatter::LONG, IntlDateFormatter::NONE );
```

Comment:
It seems to work properly with just `$book->lang` instead of `$book->getLocale()`. It would allow ICU to choose the most common locale for each language and not existing locales like `en_EN`.

Reply:
Oh yes, that works great! Thanks.